src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Reflection.Emit.cs" />
+    <Compile Include="$(CommonPath)System\Obsoletions.cs" Link="Common\System\Obsoletions.cs" />
```

Comment:
Same here.

Reply:
Updated in https://github.com/dotnet/runtime/pull/39261/commits/29f28769f25628bf532a382f6d7f9e89419e5413